apps/portal/extensions/locales/locale_si.json

```diff
@@ -139,4 +139,6 @@
   "designer.componentProperties.priority": "පූර්වතාව",
   "bad.request" : "වැරදි ඉල්ලීමකි",
   "forbidden" : "Forbiden"
```

Comment:
Fixed in commit 2c68f18dc9b0dc8bfdba57b8f658b2f55d199e99
https://github.com/wso2/carbon-dashboards/pull/452/commits/2c68f18dc9b0dc8bfdba57b8f658b2f55d199e99#diff-b5bb95287d2a3bc59e75146a3ab74c38R147